README.md

```diff
@@ -194,6 +194,11 @@ To create an instance "default" from a remote URL (use carefully, with a trustab
 $ limactl start --name=default https://raw.githubusercontent.com/lima-vm/lima/master/examples/alpine.yaml
 ```
 
+To recreate an existing instance:
```

Comment:
Maybe it is just me, but I would want to be explicit that this action will delete the existing instance. In some ways it is implied, but I think it doesn't hurt to spell it out that all content is lost:

```suggestion
To destroy an instance and recreate it with identical settings (resets the content of the instance):
```